.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,45 @@
+name: CompatHelper
```

Comment:
One of the CI checks is failing because we have to include the apache license text at the top of every single file (like [here](https://github.com/apache/arrow-julia/blob/main/.github/workflows/TagBot.yml)).